src/main/java/com/flightstats/hub/util/HubUtils.java

```diff
@@ -116,7 +116,7 @@ public void startWebhook(Webhook webhook) {
         }
     }
 
-    public void stopGroupCallback(String groupName, String sourceChannel) {
+    public void deleteWebhook(String groupName, String sourceChannel) {
         String groupUrl = getSourceUrl(sourceChannel) + "/group/" + groupName;
         log.debug("stopping {} ", groupUrl);
```

Comment:
It looks like to me that this logger should be updated from "stopping" to "deleting" as well